src/orders/components/OrderRefundDatagrid/datagrid.test.ts

```diff
@@ -0,0 +1,85 @@
+import { grantedRefunds } from "@dashboard/orders/fixtures";
+import { GridCellKind } from "@glideapps/glide-data-grid";
+import { intlMock } from "@test/intl";
+
+import {
+  createGetCellContent,
+  orderRefundStaticColumnsAdapter,
+} from "./datagrid";
+
+const emptyColumn = {
+  id: "empty",
+  title: "",
+  width: 20,
+};
+const currentTheme = "defaultLight";
+
+describe("Order refund datagrid", () => {
+  it("getCellContent returns proper data", () => {
```

Comment:
What does it mean "Returns proper data" - what are the "proper data"? Each scenario deserves own test case, where each has descriptive name eg. "(it) presents newly created grant refund with status draft that is not yet settled" - so very narrow test case